module/Application/src/Controller/StatusController.php

```diff
@@ -121,8 +119,27 @@ public function getCurrentProcessingStatus($id)
         return $currentProcessingStatus;
     }
 
-    private function updateMetadata($lpaId, $lpaStatus)
+    public function getApplicationRejectedDate($id)
+    {
+        $lpaResult = $this->getService()->fetch($id, $this->routeUserId);
+
+        if ($lpaResult instanceof ApiProblem) {
+            $this->getLogger()->err('Error accessing LPA data: ' . $lpaResult->getDetail());
+            return $lpaResult;
+        }
+        /** @var Lpa $lpa */
+        $lpa = $lpaResult->getData();
```

Comment:
Since $lpa isn't used again perhaps these two lines could be merged?
`$metaData = $lpaResult->getData()->getMetaData();`

Reply:
amended